services/src/handlers/wcs.rs

```diff
@@ -311,18 +318,19 @@ async fn get_coverage<C: Context>(
 
     let processor = initialized.query_processor().context(error::Operator)?;
 
-    let spatial_resolution: SpatialResolution = if let Some(grid_offsets) = request.gridoffsets {
-        grid_offsets.spatial_resolution(request.gridbasecrs)?
-    } else {
-        // TODO: proper default resolution
-        SpatialResolution {
-            x: request.boundingbox.partition.size_x() / 256.,
-            y: request.boundingbox.partition.size_y() / 256.,
-        }
-    };
+    let spatial_resolution: SpatialResolution =
+        if let Some(spatial_resolution) = request.spatial_resolution() {
+            spatial_resolution?
+        } else {
+            // TODO: proper default resolution
+            SpatialResolution {
+                x: request_partition.size_x() / 256.,
+                y: request_partition.size_y() / 256.,
```

Comment:
config params?

Reply:
I am not sure. I think WCS would respond with the native resolution if none is specified.

Reply:
but there is no native resolution i the engine

Reply:
isn't it still the plan to support that one day? I can make a configuration but not sure if that makes any sense because it is constant for all zoom levels and extents. If we don't plan to make native resolutions available some day I would suggest to throw an error if no resolution is specified.